examples/benchmarks/bubblebench/baseline/.eslintrc.cjs

```diff
@@ -5,5 +5,8 @@
 
 module.exports = {
 	extends: [require.resolve("@fluidframework/eslint-config-fluid"), "prettier"],
-	rules: {},
+	rules: {
+		// AB#18875
```

Comment:
Can you also add an explanation of the TODO, in addition to the work item number? That way folks without access to our azure devops still have context.

Reply:
We should also probably prefix these with "TODO" for clarity.